definitions.md

```diff
@@ -141,10 +147,20 @@ A piece is a portion of a file that gets fitted into a sector.
 
 ****The initial commitment of a storage miner to provide a number of sectors to the system.
 
+#### Polling Time
+
+Security Parameter. Number of rounds between every proving period.
```

Comment:
this is a bit misleading since the proving period is the duration of an entire post.

Polling time is the time between two online proofs of replication in a PoSt proof.

We can be more specific to what it means in proof of space, but this will be ok for spec purposes